File: Scripts/Models/Status.js
const Status = class {
    static PENDING = 1
    static IN_PROGRESS = 2
    static NEED_INFO = 3
    static COMPLETED = 4
    static WAITING = 5
    static WONT_DO = 6

    constructor (
        id,
        name,
    ) {
        this.id = id
        this.name = name
    }
}

module.exports = { Status }

File: ProTracker.Backend/ProTracker.Models/Status.cs
namespace ProTracker.Models;

public class Status
{
    public const int PENDING = 1;
    public const int IN_PROGRESS = 2;
    public const int NEED_INFO = 3;
    public const int COMPLETED = 4;
    public const int WAITING = 5;
    public const int WONT_DO = 6;

    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
}
